source/_components/switch.snmp.markdown

```diff
@@ -33,24 +33,61 @@ Configuration variables:
 - **command_oid** (*Optional*): The SNMP OID which to set in order to turn the switch on and off, if different from `baseoid`.
 - **host** (*Optional*): The IP/host which to control. Defaults to `localhost`.
 - **port** (*Optional*): The port on which to communicate. Defaults to `161`.
-- **community** (*Optional*): community string to use for authentication. Defaults to `private`.
-- **version** (*Optional*): SNMP version to use - either `1` or `2c`. Defaults to `1`.
+- **community** (*Optional*): community string to use for authentication (SNMP v1 and v2c). Defaults to `private`.
+- **username** (*Optional*): Username to use for authentication. Defaults to empty string.
+- **auth_key** (*Optional*): Authentication key to use for SNMP v3. Defaults to no key.
+- **auth_protocol** (*Optional*): Authentication protocol to use for SNMP v3. Defaults to `none`.
+- **priv_key** (*Optional*): Privacy key to use for SNMP v3. Defaults to no key.
+- **priv_protocol** (*Optional*): Privacy protocol to use for SNMP v3. Defaults to `none`.
+- **version** (*Optional*): SNMP version to use - either `1`, `2c` or `3`. Defaults to `1`.
```

Comment:
It would be nice you could update this to the [new style](https://www.home-assistant.io/developers/documentation/create_page/#configuration).

No requirement or blocker, just me friendly asking :wink: